### DISCUSSION TITLE
Re enabling after a kwin update without a build?

### DISCUSSION THREAD

**Discussion by Kenny-Dave** [matinlotfali/KDE-Rounded-Corners#281] [Q&A]
Thanks for this, it's great. Normal corners are very jarring. 

Anyway: is it possible to re-enable autorun if I've installed from a deb, not built from source?

**Comment by matinlotfali** [comment]
Sadly, when plasma and Kwin update, the effect must also be rebuilt/recompiled.

In the case of a deb install, one needs to redownload the latest deb package (they are built every day) and install it.

Now I understand that the above can also be automated in a script and executed at every login, but the fact that a script would download a deb package and install it automatically is a big security risk.

That is why we might benefit more if we can have package repositories instead so the package manager can do the dirty work for us.

**Comment by matinlotfali** [comment]
Gladly, Copr repo helped me with the automatic updates. I need to find something similar for deb packages.
